vllm/model_executor/models/config.py

```diff
@@ -388,7 +398,7 @@ def verify_and_update_config(cls, vllm_config: "VllmConfig") -> None:
                 or cache_config.block_size < attn_block_size):
             cache_config.block_size = attn_block_size
             logger.info(
-                "Setting attention block size to %d tokens "
+                "Setting attention physical block size to %d tokens "
```

Comment:
I think we don't need this line. We can add another logger.info in an appropriate place when this PR is enabled.